test/gtest/uct/ib/test_cq_moderation.cc

```diff
@@ -62,33 +75,33 @@ class test_uct_cq_moderation : public uct_test {
         m_sender->destroy_ep(0);
     }
 
-    void iface_arm(uct_iface_h iface) {
-        struct pollfd pfd;
-        int fd;
+    static void async_event_handler(void *arg, unsigned flags) {
```

Comment:
maybe inherit this class from  test_uct_ib and reuse common stuff?
also it looks quite common to be moved even to uct_test. Then create_entity calls can just use default values from uct_test base

Reply:
unfortunately, it is hard to inherit - different number of event context used
so, each test init/allocate async event contexts will use the needed number of ctxs